src/tests/functional/drive_tests.js

```diff
@@ -19,16 +20,17 @@ test("test drive to external link", async ({ page }) => {
     await route.fulfill({ body: "Hello from the outside world" })
   })
 
-  page.click("#drive_enabled_external")
+  await page.click("#drive_enabled_external")
   await nextBody(page)
 
   assert.equal(await page.evaluate(() => window.location.href), "https://example.com/")
   assert.equal(await page.textContent("body"), "Hello from the outside world")
 })
 
 test("test drive enabled by default; click link inside data-turbo='false'", async ({ page }) => {
-  page.click("#drive_disabled")
-  await nextBody(page)
```

Comment:
What is the significance of removing the call to wait for the `nextBody`?

Reply:
The significance of the change is to simplify the code by leveraging Playwright's built-in waiting capabilities, making it more concise and still ensuring that the test waits for the expected state transitions before performing assertions. To buttress further:

1. `nextBody` waits for navigation: 
The `nextBody` function waits for the page to navigate or load a new content body. This is useful when you want to ensure that the page has fully transitioned to a new state before making assertions.

2. Replacing with await page.click: 
By using `await page.click("#drive_disabled")` instead, you are now explicitly waiting for the page to click the element with the id `drive_disabled`. Playwright's `page.click` function includes implicit waiting behavior. It will wait for the element to become clickable and for any resulting navigation. DOM changes to complete. In other words, it combines the click action with the necessary waiting for the page to update.

Having said all that, I am open to revert that line back to `await nextBody(page)` given unformnity with line 24.

Reply:
I have reverted it back to `nextBody` in another commit solely for uniformity in explicitness sake.

Reply:
Thank you for the explanation. Unfortunately, Playwrights built-in synchronization when waiting for navigations isn't effective with Turbo, since Turbo intervenes in the page navigation process and issues fetch requests for content. There are no full page navigations for Playwright to listen for. That's why manual timing synchronization with helpers like nextBody are necessary.